students/uss_tomasz/lesson_06_dictionaries_tuples_sets_args_kwargs/6_3/uppercase.py

```diff
@@ -0,0 +1,18 @@
+def capitalize(lower_case_word):
+    if not isinstance(lower_case_word, str):
+        return False
```

Comment:
Not a best idea to mix returned types this way (bool + str). You better return None or raise exception if types dont match